### PR TITLE
[1LP][RFR]Adding new test for customer BZ 1695008

### PR DESCRIPTION
**Pull request by kedark3**
^ Make sure to prefix the Title with [WIP]/[WIPTEST]

Purpose or Intent
=================

__Adding tests__ for VMware customer bzs.

### CODE REVIEW

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 1]
```diff
@@ -278,3 +278,36 @@ def test_vmware_provisioned_vm_host_relationship():
             3.CFME Provisioned VM should have host relationship.
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_esxi_reboot_not_orphan_vms():
+    """
+    By mimicing ESXi reboot effect on VMs in CFME, make sure they are not getting marked orphaned.
```
Comment (john-dupuy) [inline cfme/tests/infrastructure/test_vmware_provider.py]: Small typo -> "mimicking"

Comment (kedark3) [reply cfme/tests/infrastructure/test_vmware_provider.py]: Done

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 2]
```diff
@@ -278,3 +278,36 @@ def test_vmware_provisioned_vm_host_relationship():
...
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: critical
+        initialEstimate: 1/2h
+        testtype: integration
+        testSteps:
+            1.Add VMware provider to CFME
+            2.SSH to CFME appliance and perform following steps in rails console
+                '''
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_vmware_provider.py]: Q: why do you need these quotes here?

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 3]
```diff
@@ -278,3 +278,36 @@ def test_vmware_provisioned_vm_host_relationship():
...
+                puts "ID [#{vm.id}] ems_ref [#{vm.ems_ref}]"
+                vm.update_attributes(:uid_ems => SecureRandom.uuid)
+                '''
+            3.Refresh the provider
+        expectedResults:
+            1.Provider added successfully and is refreshed
+            2.VM's uid_ems is modified
+            3.VM is still active and usable in cloudforms, not archived/orphaned.
+    """
+    pass
```
Comment (izapolsk) [inline cfme/tests/infrastructure/test_vmware_provider.py]: when you have docstring, you don't need to put pass.

**Comment by izapolsk** [comment]
@kedark3, your PR is ready for merge but please look thru and fix comments above

File: cfme/tests/infrastructure/test_vmware_provider.py (hunk 3 above)
Comment (jawatts) [reply cfme/tests/infrastructure/test_vmware_provider.py]: ^ This was discussed in the reviewers call, for now we are keeping `pass`.

File: cfme/tests/infrastructure/test_vmware_provider.py (hunk 2 above)
Comment (kedark3) [reply cfme/tests/infrastructure/test_vmware_provider.py]: I talked to @jawatts yesterday and he said that was okay. These are rails commands and I wanted to make sure they are treated as a block within this step. Justin do you think its ok to keep it ?

Comment (jawatts) [reply cfme/tests/infrastructure/test_vmware_provider.py]: @kedark3 I think it is okay to keep it.

One question after reading this PR, couldn't this be automated fairly quickly?

**Pull request merged**